Dockerfile

```diff
@@ -15,10 +19,11 @@ COPY public public
 COPY service service
 COPY util util
 COPY web web
-COPY index.js index.js
-COPY express.js express.js
-COPY sequelize.js sequelize.js
+COPY index.ts index.js
+COPY express.ts express.ts
+COPY sequelize.ts sequelize.ts
 
 ENV NODE_ENV production
 
-CMD ["node", "index.js"]
\ No newline at end of file
+ENTRYPOINT ["/usr/bin/dumb-init", "--"]
+CMD ["node", "index.ts"]
```

Comment:
Please add a newline at the end of the file... Hint: Can be forced via visualstudio code config.

Reply:
Thanks, found the config entry. Should be in any new commits from now on